ci/scripts/build-presto-package.sh

```diff
@@ -63,50 +63,81 @@ rm -rf ${package_name} ${package_name}.tar.gz
 mkdir -p ${package_name}/lib
 mkdir -p ${package_name}/function
 mkdir -p ${package_name}/bin
-mkdir -p ${package_name}/archive
-cp ./presto/presto-native-execution/_build/release/presto_cpp/function/RuntimeFunctions.bc ./${package_name}/function
+cp -a ./presto/presto-native-execution/_build/${PRESTO_CPP_MODE}/presto_cpp/function/RuntimeFunctions.bc ./${package_name}/function
+cp -a ./presto/presto-native-execution/_build/${PRESTO_CPP_MODE}/presto_cpp/main/presto_server ./${package_name}/bin
 
-DEPS_LIBRARY=(
-    ./presto/presto-native-execution/presto_cpp/main/lib/libcider.so
-    /usr/local/lib/libantlr4-runtime.so
-    /lib/x86_64-linux-gnu/libre2.so
-    /lib/libprotobuf.so
+DEPS_LIBRARY=( 
+    ./presto/presto-native-execution/presto_cpp/main/lib/libcider.so*
+    /usr/local/lib/libantlr4-runtime.so.4.9.3
     /usr/local/lib/libLLVM-9.so
-    /usr/local/lib/libtbb.so
-    /usr/local/lib/libhdfs3.so
-    /usr/lib/x86_64-linux-gnu/libgsasl.so
-    /usr/lib/x86_64-linux-gnu/libntlm.so
-    /lib/x86_64-linux-gnu/libdouble-conversion.so
-    /lib/x86_64-linux-gnu/libglog.so
-    /lib/x86_64-linux-gnu/libgflags.so
-    /lib/x86_64-linux-gnu/libsodium.so
-    /lib/x86_64-linux-gnu/libssl.so
-    /lib/x86_64-linux-gnu/libcrypto.so
-    /lib/x86_64-linux-gnu/libboost_context.so
-    /lib/x86_64-linux-gnu/libevent-2.1.so
-    /lib/x86_64-linux-gnu/libsnappy.so
-    /lib/x86_64-linux-gnu/libboost_regex.so
-    /lib/x86_64-linux-gnu/libboost_locale.so
-    /lib/x86_64-linux-gnu/libboost_log.so
-    /lib/x86_64-linux-gnu/libboost_filesystem.so
-    /lib/x86_64-linux-gnu/libboost_program_options.so
-    /lib/x86_64-linux-gnu/libboost_thread.so
-    /lib/x86_64-linux-gnu/libedit.so
-    /lib/x86_64-linux-gnu/libunwind.so
-    /lib/x86_64-linux-gnu/libicui18n.so
-    /lib/x86_64-linux-gnu/libicuuc.so
-    /lib/x86_64-linux-gnu/libbsd.so
-    /lib/x86_64-linux-gnu/libicudata.so
+    /usr/local/lib/libtbb.so.12
+    /usr/local/lib/libhdfs3.so.1
+    /lib/x86_64-linux-gnu/libglog.so.0
+    /lib/x86_64-linux-gnu/libgflags.so.2.2
+    /lib/x86_64-linux-gnu/libdouble-conversion.so.3
+    /lib/x86_64-linux-gnu/libsodium.so.23
+    /lib/x86_64-linux-gnu/libssl.so.1.1
```

Comment:
why add these？

Reply:
The ubuntu20.04 container requires these dependencies